changes/1519.fixed

```diff
@@ -0,0 +1 @@
+Extending the model table columns thant need to display copy button when hover
```

Comment:
```suggestion
Extending the model table columns that need to display copy button when hovered over.
```

Reply:
@HanlinMiao lol. thank you! I fixed it now.